sources/editor/Xenko.Core.Assets.Editor/ViewModel/EditorViewModel.cs

```diff
@@ -227,11 +235,47 @@ private async Task OpenWebPage(string url)
             }
         }
 
+        public void RemoveRecentFile(UFile filePath)
+        {
+            var packageVersion = PackageSessionHelper.GetPackageVersion(filePath);
+            //Remove considering old projects that have been deleted or upgraded from older versions
+            if (packageVersion == null || string.Compare(packageVersion.ToString(), "3.0", StringComparison.Ordinal) <= 0)
```

Comment:
Not sure the check for version 3 makes sense. Previous versions can still be installed.

Reply:
This check is not a block for old versions, it is only for executing extra codes. Older packages that have been added / removed / deleted by xenko versions <= 3.0 need to be handled differently.